crates/services/p2p/src/p2p_service.rs

```diff
@@ -461,16 +461,9 @@ impl<Codec: NetworkCodec> FuelP2PService<Codec> {
                     }
                     return Some(FuelP2PEvent::PeerDisconnected(peer_id))
                 }
-                PeerInfoEvent::TooManyPeers {
-                    peer_to_disconnect,
-                    peer_to_connect,
-                } => {
+                PeerInfoEvent::TooManyPeers { peer_to_disconnect } => {
                     // disconnect the surplus peer
                     let _ = self.swarm.disconnect_peer_id(peer_to_disconnect);
-                    // reconnect the reserved peer
```

Comment:
We already have a `health` timer that connects to disconnected reserved peers. We don't need to duplicate this work here=)